test/parse/assert/assertreturn.txt

```diff
@@ -1,8 +1,10 @@
 ;;; FLAGS: --spec
 (module
-  (func $foo (result i32) (i32.const 0))
+  (func $foo (result i32) 
```

Comment:
this is fine, but it's OK to keep these as a single line too:
(func $foo (result i32) i32.const 0)